File: Tipper/app/src/main/java/com/example/tipper/HomeScreen.java
package com.example.tipper;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class HomeScreen extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home_screen);

        Button calculateBMIButton = findViewById(R.id.calculate_button);
        Button quizButton = findViewById(R.id.quiz_button);

        calculateBMIButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(HomeScreen.this, MainActivity.class));
            }
        });
        quizButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(HomeScreen.this, QuizActivity.class));
            }
        });

    }

}

File: kotlin/app/src/main/java/com/example/tipper/HomeScreen.kt
package com.example.tipper

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.tipper.R
import android.content.Intent
import android.widget.Button
import com.example.tipper.MainActivity
import com.example.tipper.QuizActivity

class HomeScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home_screen)
        val calculateBMIButton = findViewById<Button>(R.id.calculate_button)
        val quizButton = findViewById<Button>(R.id.quiz_button)
        calculateBMIButton.setOnClickListener {
            startActivity(
                Intent(
                    this@HomeScreen,
                    MainActivity::class.java
                )
            )
        }
        quizButton.setOnClickListener {
            startActivity(
                Intent(
                    this@HomeScreen,
                    QuizActivity::class.java
                )
            )
        }
    }
}